tests/publiccloud/slem_basic.pm

```diff
@@ -21,7 +21,7 @@ sub run {
     select_serial_terminal();
     my $provider = $self->provider_factory();
     $provider->{username} = 'suse';
-    my $instance = $self->{my_instance} = $provider->create_instance();
+    my $instance = $self->{my_instance} = $provider->create_instance(check_guestregister => 0);
```

Comment:
@jlausuch  FYI

Reply:
for that, [3755](http://10.168.4.117/tests/3755) |PASS| Micro-5-3 BYOS GCE / no-regression test